index.html

```diff
@@ -3951,41 +3951,191 @@ <h2>Syntaxes</h2>
         <h3>JSON-LD</h3>
 
         <p>
+The world of credentials is vast and varied. A physical wallet places
+relatively few constraints on the shape and size of those credentials–and it
+places no restriction on their contents. Likewise, Verifiable Credential Data
+Model’s use of [[!JSON-LD]] provides the consistent shape for all compliant
+Verifiable Credentials, so that digital wallets can store credentials with
+widely varying content, providing value to a plethora of communities, and
+enabling many diverse use cases–from coupons to authentication.
+        </p>
+
+        <p>
+Consequently, Wallet providers can have the confidence that any Verifiable
+Credential will “fit” while Issuers and Verifiers benefit from a common
+exchange representation that provides built in extensibility without the
+high costs from centralized terminology registration or JSON format churn.
+        </p>
+
+        <p>
+Issuers can add claims terminology unique to their communities via [[!JSON-LD]]’s
+context files rather than be dependent on a single, centralized registry
+system. This webby decentralized approach provides infinite scalability. Future
+standardization efforts, then focus on vocabulary definitions and
+object-oriented modeling provided by communities of practice vs. idiosyncratic
+JSON format creation for individual use cases or scenarios.
+        </p>
+
+        <p>
+Verifiers can furthermore lean on the similarly provided proof mechanisms
+without writing new document parsers for each new security mechanism.
+        </p>
+
+        <p>
+Lastly, Wallets can then move these consistently shaped credentials between
+Issuers and Verifiers without the need to create new storage systems for each
+format with the confidence that each relying party can depend on consistent
+grammatical structures and inbuilt object-oriented data modeling.
+        </p>
+
+        <p>
+The Verifiable Credential Data Model’s consistent object-oriented modeling
+constraints are familiar to developers and enable better reuse and composition
+of data. That modeling is built from clean expressions of subject, property,
+values which comprise the shape of each atomic claim within a VCDM credential.
+This approach empowers Wallets to store any credential shaped in this way (as
+with physical wallets) and to index the contained claims made across all the
+holder’s credentials such that credentials may be quickly and easily found and
+presented when needed.
```

Comment:
```suggestion
The consistent object-oriented modeling constraints of the Verifiable
Credential Data Model (VCDM) are familiar to developers and enable better
reuse and composition of data. That modeling is built from clean expressions
of _subjects, properties, and values_ (also referred to as _entities,
attributes, and values_, _E-A-V_, as well as _subjects, predicates, and
objects_, _S-P-O_,) which comprise the shape of each atomic claim within
a VCDM credential. This approach empowers Wallets to store any credential
shaped in this way (as with physical wallets) and to index the contained
claims across all the holder's credentials, such that credentials may be
quickly and easily found and presented when needed.
```

Reply:
Added with credit in the commit message. 😄